ui/app/adapters/secret-engine.js

```diff
@@ -87,10 +87,8 @@ export default ApplicationAdapter.extend({
     }
   },
 
-  findRecord(store, type, path, snapshot) {
-    if (snapshot.attr('type') === 'ssh') {
-      return this.ajax(`/v1/${encodePath(path)}/config/ca`, 'GET');
-    }
+  findRecord() {
```

Comment:
Out of curiosity why do we need this now as it just returns an empty object 🤔

Reply:
We don't, but I was being conservative and only wanted to remove all things SSH related. I had removed it and then thought, well it's technically out of scope so put it back in.

Reply:
It seems reasonable to remove it if it's cleanup that's a side effect of this work. Especially if it doesn't have a function now 🤷